src/OpenTelemetry/Logs/LogRecord.cs

```diff
@@ -0,0 +1,36 @@
+// <copyright file="LogRecord.cs" company="OpenTelemetry Authors">
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// </copyright>
+
+#if NETSTANDARD2_0
+using System;
+using Microsoft.Extensions.Logging;
+
+namespace OpenTelemetry.Logs
+{
+    /// <summary>
+    /// Log record base class.
+    /// </summary>
+    public struct LogRecord
+    {
+        public DateTime Timestamp;
```

Comment:
CodeAnalysis would warn you not to make public fields here. Any reason not to add `{ get; set; }`?

Reply:
The only reason is my lack of C# knowledge 😄, fixed it.